internal/csi-common/server.go

```diff
@@ -97,7 +97,7 @@ func (s *nonBlockingGRPCServer) serve(endpoint string, srv Servers) {
 		klog.Fatalf("Failed to listen: %v", err)
 	}
 
-	server := grpc.NewServer()
+	server := grpc.NewServer(NewMiddlewareServerOption(false))
```

Comment:
we can remove `metrics` argument from `NewMiddlewareServerOption` as it always set to false but it can be in separate PR.

Reply:
See #4264